main/getting-started/Documentation-Map.md

```diff
@@ -0,0 +1,65 @@
+# Agoric Documentation Map
+
+Agoric provides three different types of documentation. This page
+explains what they are and when you should use them, as well as
+what order to read them.
+
+Our three documentation types are:
+- **Getting Started**: These are the first you should
+read. They cover what you need to do to install, set up, and start
+using Agoric's smart contract tools and introduce those tools and
+their underlying concepts and designs.
+- **Guides**: These are more detailed and complete descriptions
+of our tools and their concepts, designs, and usage. They include
+descriptions and examples of how to do various tasks with Agoric
+software.
+- **Reference**: These are the specs for our APIs and commands,
+  as well as procedure checklist tables that specify and explain the
+  steps for extended procedures, such as installing Agoric
+  software.
+
+Some documents fall into multiple categories. For example, our Agoric
+installation document is both a Reference and a Getting Started document.
+
+You should read them in this order:
+1. **Agoric Prerequisites**: What you need to do before installing
+   Agoric software and installing the Agoric SDK.
+2. **Introduction to ERTP**: A Getting Started document that
+  briefly explains the concepts and components of Agoric's Electronic
+  Rights Transfer Protocol (ERTP) which is the base of Agoric
+  software.
+3. **Introduction to Zoe**: A Getting Started document that
+  briefly explains the concepts and components of Zoe, our platform
+  for writing smart contracts in JavaScript. Zoe implements our smart
+  contract guarantee that users either get what they wanted from the
+  contract or a full refund of what they put up for it. Regardless of
+  whether the contract is buggy or malicious.
```

Comment:
==> "or a full refund of what they put up for it, regardless of whether the contract is honest, buggy or malicious."